addons/website_project/tests/test_website_project.py

```diff
@@ -53,7 +53,10 @@ def test_admin_task_submission(self):
         self.assertTrue(task.exists())
         self.assertEqual(task.partner_id, self.test_partner, "Partner id should not be False")
         self.assertFalse(task.email_cc, "email_cc field should be empty")
-        self.assertIn('This Task was submitted by Mitchell Admin (admin@yourcompany.example.com) on behalf of test@partner.com', html2plaintext(task.description), "Warning message should be displayed in description of task")
-
-        mail_message = task.message_ids.filtered(lambda m: m.body == '<div class="alert alert-info">This Task was submitted by Mitchell Admin (admin@yourcompany.example.com) on behalf of test@partner.com</div>')
+        if self.env['res.users'].browse(2).name == "Administrator":
+            asserttext = 'Administrator (admin@example.com)'
+        elif self.env['res.users'].browse(2).name == "Mitchell Admin":
+            asserttext = 'Mitchell Admin (admin@yourcompany.example.com)'
```

Comment:
why not doing `self.env.ref('base.user_admin')` to get the admin user? 🤷🏼‍♂️ 

By doing that we will no longer need to check the user name to know in which config we are when we launch the test. 🙂